File: Weather/src/main/java/com/aliucord/plugins/weather/WeatherResponse.java
package com.aliucord.plugins.weather;

public class WeatherResponse {
    public Condition[] current_condition;
    public static class Condition {
        public String FeelsLikeC;
        public String FeelsLikeF;
        public String cloudcover;
        public String humidity;
        public String temp_C;
        public String temp_F;
        public String uvIndex;
        public WeatherDesc[] weatherDesc;
        public String winddir16Point;
        public String windspeedKmph;
        public String windspeedMiles;
    }
    public static class WeatherDesc {
        public String value;
    }

    public NearestArea[] nearest_area;
    public static class NearestArea {
        public AreaName[] areaName;
        public Country[] country;
        public Region[] region;
    }
    public static class AreaName {
        public String value;
    }
    public static class Country {
        public String value;
    }
    public static class Region {
        public String value;
    }
}

File: Weather/src/main/java/com/aliucord/plugins/weather/WeatherResponse.kt
package com.aliucord.plugins.weather

class WeatherResponse(
    val current_condition: Array<Condition>,
    val nearest_area: Array<NearestArea>
) {
    class Condition (
        val FeelsLikeC: String,
        val FeelsLikeF: String,
        val cloudcover: String,
        val humidity: String,
        val temp_C: String,
        val temp_F: String,
        val uvIndex: String,
        val weatherDesc: Array<WeatherDesc>,
        val winddir16Point: String,
        val windspeedKmph: String,
        val windspeedMiles: String
    )

    class WeatherDesc (
        val value: String
    )

    class NearestArea (
        val areaName: Array<AreaName>,
        val country: Array<Country>,
        val region: Array<Region>
    )

    class AreaName (
        val value: String
    )

    class Country (
        val value: String
    )

    class Region (
        val value: String
    )
}